dune-project

```diff
@@ -34,9 +34,9 @@
   "ReasonReact helps you use Reason to build React components with deeply integrated, strong, static type safety.\n\nIt is designed and built by people using Reason and React in large, mission critical production React codebases.")
  (depends
   (ocaml
-   (>= 4.13.0))
+   (and (>= 5.1.0) (< 5.2.0)))
```

Comment:
I wouldn't add upper bounds to the compiler, specially considering that Melange already has its own constraints for it.